### PR TITLE
:memo: Fixed grammatical errors

### PR DESCRIPTION
**Pull request by pranjaljain0**
Page: https://flutter.dev/docs/perf/app-size
Page Location: https://github.com/flutter/website/edit/master/src/docs/perf/app-size.md

### CODE REVIEW

File: src/docs/perf/app-size.md [hunk 1]
```diff
@@ -13,17 +13,17 @@ and it may break the limit of useful features like Android instant apps.
 
 By default, launching your app with `flutter run`,
 or by clicking the **Play** button in your IDE
-(as used in [Test drive][] and
+(as used in a [Test drive][] and
```
Comment (kwalrath) [inline src/docs/perf/app-size.md:16]:
```suggestion
(as used in [Test drive][] and
```

File: src/docs/perf/app-size.md [hunk 2]
```diff
@@ -13,17 +13,17 @@ and it may break the limit of useful features like Android instant apps.
...
 The app size of a debug build is large due to
 the debugging overhead that allows for hot reload
-and source level debugging. As such, it is not representative of a production
+and source-level debugging. As such, it is not representative of a production
 app end users download.
 
 ## Checking the total size
 
-A default release build, such as one created by `flutter build apk` or
+A default release builds, such as one created by `flutter build apk` or
```
Comment (kwalrath) [inline src/docs/perf/app-size.md:26]:
```suggestion
A default release build, such as one created by `flutter build apk` or
```

File: src/docs/perf/app-size.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ View the application's download and install size in the **Android vitals** ->
 {% include app-figure.md image="perf/vital-size.png" alt="App size tab in Google Play Console" %}
 
 The download size is calculated based on an XXXDHPI (~640dpi) device on an
-arm64-v8a architecture. Your end users' download sizes may vary depending on
+arm64-v8a architecture. Your end-users' download sizes may vary depending on
```
Comment (kwalrath) [inline src/docs/perf/app-size.md:59]:
```suggestion
arm64-v8a architecture. Your end users' download sizes may vary depending on
```

Per https://developers.google.com/style/word-list#letter-e, we use "user" or "end user" for the noun form.

File: src/docs/perf/app-size.md [hunk 4]
```diff
@@ -110,15 +110,15 @@ section in the Flutter [FAQ][].
 
 ## Breaking down the size
 
-Starting in Flutter version 1.22 and in DevTools starting in version 0.9.1,
+Starting in Flutter version 1.22 and DevTools starting in version 0.9.1,
```
Comment (kwalrath) [inline src/docs/perf/app-size.md:113]:
```suggestion
Starting in Flutter version 1.22 and DevTools version 0.9.1,
```

File: src/docs/perf/app-size.md [hunk 5]
```diff
@@ -110,15 +110,15 @@ section in the Flutter [FAQ][].
...
 a size analysis tool is included to help developers understand the breakdown
 of the release build of their application.
 
 {{site.alert.warning}}
   As stated in the [checking total size](#checking-the-total-size) section
-  above, an upload package is not representative of your end users' download
+  above, an upload package is not representative of your end-users' download
```
Comment (kwalrath) [inline src/docs/perf/app-size.md:119]:
```suggestion
  above, an upload package is not representative of your end users' download
```

File: src/docs/perf/app-size.md [hunk 6]
```diff
@@ -110,15 +110,15 @@ section in the Flutter [FAQ][].
...
   size. Be aware that redundant native library architectures and asset densities
-  seen in the breakdown tool can be filtered by the Play Store and App Store.
+  are seen in the breakdown tool that can be filtered by the Play Store and App Store.
```
Comment (kwalrath) [inline src/docs/perf/app-size.md:121]:
```suggestion
  seen in the breakdown tool can be filtered by the Play Store and App Store.
```

**Review by kwalrath — CHANGES_REQUESTED** [review]
Some good changes; some others I didn't agree with.

**Review by pranjaljain0 — COMMENTED** [review]
Requested changes have been made,
Thank you :)

**Review by kwalrath — APPROVED** [review]
LGTM!

**Pull request merged**